### DISCUSSION TITLE
Design for use?

### DISCUSSION THREAD

**Discussion by Aqshola** [mgilangjanuar/teledrive#9] [Ideas]
should we copy google drive layout for dashboard, login and others? also using [Heroicon](https://heroicons.com/) for icons?

**Comment by mgilangjanuar** [comment]
Good question 🤔

We can start from the simple form and basic list for files and tabs for categorized it (Photo, Video, PDF, txt, etc), with 1 big float action button for upload, I think 🤔

I'll share the mockup for visualize it.

**Comment by mgilangjanuar** [comment]
<img width="1576" alt="Screen Shot 2021-08-28 at 14 03 27" src="https://user-images.githubusercontent.com/3146378/131209725-fbe5bc17-be2e-4730-b611-4a87f4c7e11c.png">

Any feedback? 🤣 

Anw it's okay to start from simple, so we can iterate the development to be more cooler 😎

**Reply by Aqshola** [reply]
Clear enough for me ahaha, at least we have layout for now

**Reply by naufaldi** [reply]
FYI, there are Chakra component for free, maybe if suits we can use it? [Chakra Component](https://chakra-templates.dev)

**Reply by mgilangjanuar** [reply]
Whoaa, let's use the template then 🤣
